### ISSUE TITLE
urlSuffix can't be defined dynamically

### ISSUE THREAD

**Issue by hackel** [getsentry/sentry-webpack-plugin#235]
I've been struggling to set up source maps for some time, and I believe it has come down to the fact that I cannot set the correct `urlSuffix` for each source file.  I'm using https://github.com/JeffreyWay/laravel-mix to configure Webpack.  For versioning/cache-busting, it adds a query string to each file (first 20 characters of the file's md5 hash), and stores these in a mix-manifest.json file.  (e.g. `"/js/app.js.map?id=c383c0e84a279a2e9843"`)  This file doesn't exist yet when I am configuring SentryWebpackPlugin since it depends on the final output file.  I'm versioning multiple files, including a separate vendor.js, so I cannot use the same suffix for everything.

Basically, I need a way to pass in a closure when configuring the plugin that will get run for *each* file to determine the correct urlSuffix.  Ideally I could just do something like `urlSuffix: filename => manifest[filename]`.  If there is a better way to accomplish this that I'm not seeing, please let me know.

Of course, it would be a lot easier if I could just configure Sentry to ignore the query string when matching up sourcemaps.

**Comment by alekbless** [comment]
Did you get sourceMaps to Sentry working when using laravel-mix?
